Bottom: e7412cf9311d0ae7bc4a6bbd9d90cf05a19dc79f
Top:    ef2e07a954903504c23ea8125195e4609efdd83c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:56:10 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 6e8c5d3..9f1d695 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2698,7 +2698,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
